Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -1366,6 +1371,32 @@ bool NetworkProcess::privateClickMeasurementEnabled() const
     return m_privateClickMeasurementEnabled;
 }
 
+void NetworkProcess::notifyMediaStreamingActivity(bool activity)
+{
+#if PLATFORM(COCOA)
+    static const char* notifyMediaStreamingName = "com.apple.WebKit.mediaStreamingActivity";
+
+    uint32_t status = NOTIFY_STATUS_FAILED;
+    if (m_mediaStreamingActivitityToken == NOTIFY_TOKEN_INVALID) {
+        status = notify_register_check(notifyMediaStreamingName, &m_mediaStreamingActivitityToken);
+        if (status != NOTIFY_STATUS_OK || m_mediaStreamingActivitityToken == NOTIFY_TOKEN_INVALID)
+            RELEASE_LOG_ERROR(IPC, "notify_register_check() for %s failed with status (%d) 0x%X", notifyMediaStreamingName, status, status);
+    }
```

Comment:
We could early return here in case of error so that we do not need the next if check.